environments/custom/playbook-lvm.yml

```diff
@@ -1,5 +1,14 @@
+# https://docs.debops.org/en/stable-3.1/ansible/roles/lvm/defaults/main.html#volume-groups-and-logical-volumes
 - name: Manage LVM volumes
   hosts: generic
+  gather_facts: yes
   become: true
-  roles:
-    - role: debops.debops.lvm
+  tasks:
+    - name: Manage LVM Logical Volumes
+      community.general.lvol:
+        lv:         '{{ item.lv }}'
+        vg:         '{{ item.vg }}'
+        size:       '{{ item.size }}'
+        force:      '{{ item.force | d(omit) }}'
+        state:      'present'
+      with_items: '{{ logical_volumes }}'
```

Comment:
Use loop instead of with_items. Read for further details.

Reply:
Yes, changed that and move the code to the module.

Reply:
done